README.md

```diff
@@ -98,11 +129,35 @@ connect(this.mapStateToThis, this.mapDispatchToThis)(this);
 connect(this.mapState, this.mapDispatch)((selectedState, actions) => {/* ... */});
 ```
 
-
 #### Remarks
 * The `mapStateToTarget` function takes a single argument of the entire Redux store’s state and returns an object to be passed as props. It is often called a selector. Use reselect to efficiently compose selectors and compute derived data.
 
 
+### select(key | function,[comparer]) => Observable
+
+Exposes a slice of state as an observable. Accepts either a key-path, or a selector function.
+
+If using the async pipe, you do not need to subscribe to it explicitly, but can use the angular Async pipe to observe for values.
+
+#### Arguments
+
+* `key` \(*string*): A key within the state that you want to subscribe to. 
+* `selector` \(*Function*): A function that accepts the application state, and returns the slice you want subscribe to for changes. 
+
+
+e.g:
+```JS
+this.counter$ = this.ngRedux.select(state=>state.counter);
+// or 
+this.counterSubscription = this.ngRedux
+  .select(state=>state.counter)
```

Comment:
Presumably this is the place for reselect-style functions that transform things, if needed?

Alternately you can just do

```
.select('counter').map(whateverReselection)
```